### PR TITLE
Add OPA documentation

### PR DESCRIPTION
**Pull request by acritelli**
Related to https://github.com/cloudify-cosmo/cloudify-terraform-plugin/pull/229

### CODE REVIEW

File: content/working_with/official_plugins/Orchestration/terraform.md [hunk 1]
```diff
@@ -273,6 +273,50 @@ cloudify.nodes.terraform.Module:properties.resource_config
...
+
+The operation provides a thing wrapper around running `opa exec` against the Terraform plan in JSON format.
+
+OPA is configured by setting the desired parameters in `cloudify.nodes.terraform.Module:properties.opa_config`:
+
+* `installation_source` - Location to download the OPA binary from. Defaults to `https://github.com/open-policy-agent/opa/releases/download/v0.47.4/opa_linux_amd64_static`
+* `executable_path` - Location of an existing OPA binary on the system. If used, the binary will not be downloaded from the `installation_source` and the existing binary will be used for policy evaluations.
+* `config` - OPA configuration file to override any default OPA behavior.
+* `policy_bundles` - A list of one or more zipped policy [bundles](https://www.openpolicyagent.org/docs/latest/management-bundles/) for use in the policy evaluation. Policy budnles can be local to the blueprint archive, or they can be downloaded from remote sources. The format for policy bundles is described below.
```
Comment (EarthmanT) [inline content/working_with/official_plugins/Orchestration/terraform.md:287]:
```suggestion
* `policy_bundles` - A list of one or more zipped policy [bundles](https://www.openpolicyagent.org/docs/latest/management-bundles/) for use in the policy evaluation. Policy bundles can be local to the blueprint archive, or they can be downloaded from remote sources. The format for policy bundles is described below.
```

Comment (EarthmanT) [reply content/working_with/official_plugins/Orchestration/terraform.md:287]: Is it possible also, to put the `policy_bundles` key indented beneath `config` in markdown?

File: content/working_with/official_plugins/Orchestration/terraform.md [hunk 2]
```diff
@@ -273,6 +273,50 @@ cloudify.nodes.terraform.Module:properties.resource_config
     * api_key: api key from Infracost, instructions how to retrieve it give in a [link](https://www.infracost.io/docs/)
     * enable: boolean, In order for it to work, must mark True.
 
+### OPA
+
```
Comment (EarthmanT) [inline content/working_with/official_plugins/Orchestration/terraform.md:277]: We should say which version of tf plugin this becomes available. (0.19.14)

**Review by EarthmanT — CHANGES_REQUESTED** [review]

File: content/working_with/official_plugins/Orchestration/terraform.md (hunk 1 above)
Comment (acritelli) [reply content/working_with/official_plugins/Orchestration/terraform.md:287]: I'm not sure what you mean. In the example below, it's indented beneath `opa_config`. It's a key on `opa_config`, not `config`.

**Pull request merged**